redirects.json

```diff
@@ -843,5 +843,10 @@
     "source": "/advanced/passkeys/supported-networks",
     "destination": "/advanced/smart-account-supported-networks?module=Safe+Passkey+Module",
     "permanent": true
+  },
+  {
+    "source": "/core-api/transaction-service-reference",
```

Comment:
To keep redirects organized, can you move it to the `/core-api` "section" instead of adding it at the end of the file?